definitions/dfe_analytics_dataform.js

```diff
@@ -1076,6 +1081,16 @@ dfeAnalyticsDataform({
                     dataType: "boolean",
                     description: "",
                 },
+                {
+                    keyName: "consent_method",
+                    dataType: "string",
+                    description: "",
+                },
+                {
+                    keyName: "unsigned_consent_document_generated",
+                    dataType: "boolean",
+                    description: "",
+                },
```

Comment:
This field no longer exists on the qualification requests, only on the assessments.